# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/named_pipe_policy.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/named_pipe_policy.h
--- a/security/sandbox/chromium/sandbox/win/src/named_pipe_policy.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/named_pipe_policy.h	Wed Jun 12 11:10:48 2019 +0100
@@ -29,15 +29,16 @@
   // Processes a 'CreateNamedPipeW()' request from the target.
   static DWORD CreateNamedPipeAction(EvalResult eval_result,
                                      const ClientInfo& client_info,
-                                     const base::string16 &name,
-                                     DWORD open_mode, DWORD pipe_mode,
+                                     const base::string16& name,
+                                     DWORD open_mode,
+                                     DWORD pipe_mode,
                                      DWORD max_instances,
                                      DWORD out_buffer_size,
                                      DWORD in_buffer_size,
-                                     DWORD default_timeout, HANDLE* pipe);
+                                     DWORD default_timeout,
+                                     HANDLE* pipe);
 };
 
 }  // namespace sandbox
 
-
 #endif  // SANDBOX_SRC_NAMED_PIPE_POLICY_H__